styleguide/source/_patterns/03-organisms/sidebar-articles.json

```diff
@@ -0,0 +1,105 @@
+          "href": "http://www.google.com",
+          "text": "Lorem ipsum dolor sit amet."
+        },
+        "image": {
+          "alt": "alt text",
+          "src": "https://ipsumimage.appspot.com/800x600x186?l=3:2|800x600&s=36",
+          "height": "600",
+          "width": "800"
+        },
+        "video": "",
+        "related": "",
+        "small": "",
+        "paragraph": {
+          "text": "A paragraph (from the Greek paragraphos, \"to write beside\" or \"written beside\") is a self-contained unit of a discourse in writing dealing with a particular point or idea. A paragraph consists of one or more sentences. Though not required by the syntax of any language, paragraphs are usually an expected part of formal writing, used to organize longer prose."
+        },
+        "metadata": {
+          "date": "Jul 28, 2017",
+          "readtime": "5 min read"
+        }
+      },
+      {
+        "title_only": "",
+        "link": {
+          "title": "A link to Google, for example.",
+          "href": "http://www.google.com",
+          "text": "Lorem ipsum dolor sit amet."
+        },
+        "image": {
+          "alt": "alt text",
+          "src": "https://ipsumimage.appspot.com/800x600x186?l=3:2|800x600&s=36",
+          "height": "600",
+          "width": "800"
+        },
+        "video": "",
+        "related": "",
+        "small": "",
+        "paragraph": {
+          "text": "A paragraph (from the Greek paragraphos, \"to write beside\" or \"written beside\") is a self-contained unit of a discourse in writing dealing with a particular point or idea. A paragraph consists of one or more sentences. Though not required by the syntax of any language, paragraphs are usually an expected part of formal writing, used to organize longer prose."
+        },
+        "metadata": {
+          "date": "Jul 28, 2017",
+          "readtime": "5 min read"
+        }
+      },
+      {
+        "title_only": "",
+        "link": {
+          "title": "A link to Google, for example.",
+          "href": "http://www.google.com",
+          "text": "Lorem ipsum dolor sit amet."
+        },
+        "image": {
+          "alt": "alt text",
+          "src": "https://ipsumimage.appspot.com/800x600x186?l=3:2|800x600&s=36",
+          "height": "600",
+          "width": "800"
+        },
+        "video": "",
+        "related": "",
+        "small": "",
+        "paragraph": {
+          "text": "A paragraph (from the Greek paragraphos, \"to write beside\" or \"written beside\") is a self-contained unit of a discourse in writing dealing with a particular point or idea. A paragraph consists of one or more sentences. Though not required by the syntax of any language, paragraphs are usually an expected part of formal writing, used to organize longer prose."
+        },
+        "metadata": {
+          "date": "Jul 28, 2017",
+          "readtime": "5 min read"
+        }
+      }
+    ]
+  }
+}
```

Comment:
You need an empty line here.
I guess it's intentional that that the organism is all using the same view mode? This is either a problem with the testing instructions or the data here.

Reply:
That was not intentional, good catch, it should be fixed now.

Reply:
This has been fixed, thank you!